core/pom.xml

```diff
@@ -33,6 +33,10 @@
   <name>Spark Project Core</name>
   <url>http://spark.apache.org/</url>
   <dependencies>
+    <dependency>
```

Comment:
This could move next to other scala-related dependencies, to sort of be consistent with order and grouping in the parent POM.

Reply:
Done